src/carriers/zfp_portmonitor/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2016 iCub Facility
+# Authors: Nicolo' Genesio
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+list(APPEND CMAKE_MODULE_PATH ${YARP_MODULE_PATH})
+
+# probably not needed ... to be tested
+set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
+
+include(YarpPlugin)
+include(YarpInstallationHelpers)
+
+# probably not needed ... plugins should set this automatically ... to be tested
+set(YARP_FORCE_DYNAMIC_PLUGINS TRUE CACHE INTERNAL "yarp_pm_zfp is always built with dynamic plugins")
+
+yarp_configure_external_installation(yarp)
+
+yarp_prepare_plugin(zfp        TYPE ZfpMonitorObject
+                               INCLUDE zfpPortmonitor.h
+                               CATEGORY portmonitor
+                               DEPENDS "CREATE_OPTIONAL_CARRIERS;ENABLE_yarpcar_portmonitor" 
+                               )
+
+if(NOT SKIP_zfp)
+
+	find_package(ZFP REQUIRED)
+
+	if (NOT ZFP_FOUND)
+		message("-- Did not find ZFP library.")
```

Comment:
@traversaro  you were right, the call to `find_package_handle_standard_args` was missing, I added it to FindZFP.cmake and I deleted the extra check after `find_package(ZFP REQUIRED)`.
@drdanz  I think that now it is ready for the code review.